packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -825,6 +874,118 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 	return <DefaultLineItemSublabel product={ product } />;
 }
 
+export function LineItemBillingInterval( { product }: { product: ResponseCartProduct } ) {
```

Comment:
I broke down LineItemSublabelAndPrice into LineItemBillingInterval and LineItemMeta to separate concerns.

For this function my thought process was 'default to our monthly/yearly/bi/tri etc plans at the end and then early return for any special cases above the 'defaults'.